src/style.css

```diff
@@ -0,0 +1,42 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars_star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars_star:first-child {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
```

Comment:
1. you already have `background-repeat: no-repeat;` in `stars_star`
2. combine all selectors using coma
```
.stars--1 .stars_star:first-child,
.stars--2 .stars_star:nth-child(-n + 2) {
  background-image: url(images/star-active.svg);
}